oval_graph/_xml_parser_comments.py

```diff
@@ -2,38 +2,44 @@
     'oval-definitions': 'http://oval.mitre.org/XMLSchema/oval-definitions-5',
 }
 
-# rework this (use static metodes)
-
 
 class _XmlParserComments:
     def __init__(self, oval_definitions):
         self.oval_definitions = oval_definitions
 
-    def _create_dict_form_criteria(self, criteria, description):
+    def _create_dict_form_criteria(self, criteria, description=None):
         comments = dict(
-            operator='AND' if criteria.get('operator') is None else criteria.get('operator'),
-            comment=description if criteria.get('comment') is None else criteria.get('comment'),
+            operator=self._get_operator(criteria),
+            comment=self._get_comment(criteria, description),
```

Comment:
What happens if the description is None?

Reply:
Returns ```None``` or ```criterion.get('comment')```.